scapy/data.py

```diff
@@ -188,7 +188,7 @@ def load_manuf(filename):
     IP_PROTOS=load_protocols(os.environ["SystemRoot"]+"\system32\drivers\etc\protocol")
     TCP_SERVICES,UDP_SERVICES=load_services(os.environ["SystemRoot"] + "\system32\drivers\etc\services")
     # Default value, will be updated by arch.windows
-    MANUFDB = load_manuf(os.environ["ProgramFiles"] + "\\wireshark\\manuf")
+    MANUFDB = None
```

Comment:
Why are you modifying this line ? This should be in a dedicated PR if really needed.